viewer/vueapp/src/components/spigraph/Spigraph.vue

```diff
@@ -503,6 +507,22 @@ export default {
       this.loading = true;
       this.error = false;
 
+      if (this.multiviewer) {
+        var availableESCluster = this.$store.state.esCluster.availableCluster.active;
```

Comment:
For new code, I'd like to use `let`. This is something we're currently working on as a lot of the old code uses `var`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let